### DISCUSSION TITLE
Included theme shows less highlighting?

### DISCUSSION THREAD

**Discussion by rieje** [sharkdp/bat#3096] [Q&A]
EDIT: There's an open pull request, hopefully it gets merged soon: https://github.com/sharkdp/bat/pull/2994

[`--theme=TwoDark` on the left, no theme specified on the right](https://0x0.st/XEXo.png). Is the TwoDark theme supposed to have much less highlighting?

P.S. Unrelated question, but do both bat and source-highlight do the same highlighting for manpages or is one not as comprehensive?

**Comment by keith-hall** [comment]
I've marked the referenced PR for auto-merge :)

I have never heard of source-highlight so can't answer precisely as I don't know how it works. `bat` very much relies on the structure of the manpage to match what it has been told to highlight in the `.sublime-syntax` definition, so there are many edge cases where it falls short due to the unstructured (in terms of markup) nature of manpages.
